Add Volume Management (#204)
harvester/docs: docs/volume/export-volume.md

---
sidebar_position: 4
sidebar_label: Export a Volume to Image
title: ""
keywords:
- Volume
Description: Export volume to image from the Volume page.
---

# Export a Volume to Image

> After creating a volume, you can export the volume to image by the following steps:

Reviewer: ```suggestion
After creating a volume, you can export the volume to an image by following the steps below:
```

> 1. Clicking the `⋮` button and selecting the `Export Image` button.

Reviewer: ```suggestion
1. Click the `⋮` button and select the `Export Image` option.
```

> 1. Select the `Namespace` that you want to export to.

Reviewer: I fully don't understand this line. Are we exporting the volume to an image or to an image?

Author: Export the volume to an image

Fixed by:
1. Select the `Namespace` of the new image.

Reviewer: Then we should mention that explicitly, please.

Author: Thanks. Fixed in the latest commit: https://github.com/harvester/docs/commit/8e5f1c959fe6ebaa6470874a38325d884e79f64b
Maybe the browser has a cache?  Need refresh？
or the `Select the Namespace of the new image` is not clear enough?

1. Configure the `Name` of the new image.

> 1. Select an existing `Storage Class`.

Reviewer 2: ```suggestion
1. Select an existing `Storage Class`.
1. (Optional) You can download the exported image from the `Images` page by clicking the `⋮` button and selecting the `Download` option.
```

> 1. Select a existing `Storage Class`.

Reviewer: ```suggestion
1. Select an existing `Storage Class`.
```
